docker/urls.py

```diff
@@ -1,34 +1,24 @@
-"""docker URL Configuration
+# Copyright 2023 UW-IT, University of Washington
+# SPDX-License-Identifier: Apache-2.0
 
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.11/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.conf.urls import url, include
-    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
-"""
 from django.conf import settings
-from django.conf.urls import url, include
+from django.urls import path, include
+from django.contrib import admin
 
 urlpatterns = [
-    url(r'^auth/', include('oauth_provider.urls')),
-    url(r'^api/', include('spotseeker_server.urls')),
-    url(r'^', include('django_prometheus.urls')), # add here for django 1.11 compatibility
+    path('api/', include('spotseeker_server.urls')),
+    path(
+        'auth/',
+        include('oauth2_provider.urls', namespace='oauth2_provider')
+    ),
 ]
 
 if settings.DEBUG:
-    from django.contrib import admin
     from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
     urlpatterns += [
-        url(r'^admin/doc/', include('django.contrib.admindocs.urls')),
-        url(r'^admin/', include(admin.site.urls)),
+        path('admin/doc/', include('django.contrib.admindocs.urls')),
+        path('admin/', admin.site.urls),
```

Comment:
This works if you create a super user and you can see all previously made access tokens, all spots and items with their extended info and images, and all clients. This could be pretty useful for testing whether access tokens work as intended and that expired ones don't work anymore.